syntax = "proto3";
package fixbug;
option cc_generic_services  = true;
message ResultCode
{
    int32 errcode = 1;
    bytes errmsg = 2;
}
message FriendListRequest
{
    uint32 id = 1;
}
message FriendListResponse
{
    ResultCode result = 1;
    bool success = 2;
    repeated bytes friend = 3;
}
service FriendServiceRpc
{
    rpc GetFriendList(FriendListRequest) returns(FriendListResponse);
}